### ISSUE TITLE
Count total data points in a measurement

### ISSUE THREAD

**Issue by Benhgift** [influxdata/influxdb#16660]
__Proposal:__
Count all the data points in a measurement and return the value.

__Current behavior:__
This is not possible. 

__Desired behavior:__
Something like:
`SHOW DATA_POINTS COUNT FROM cpu;`

__Use case:__
I'm currently profiling usage on a large influxdb instance with thousands of measurements and billions of data points, and counting the rough size of each measurement would help determine where most of our data is coming from.

**Comment by Igor-Toropov** [comment]
I don't think there are any stats on per measurement basis.
Query with COUNT aggregator and without any GROUP BY (especially time) would do just that.
Although it would be a very heavy query, especially on cpu measurement, so splitting with multiple WHERE conditions would help.
Personally I prefer counting something of interest while reading from message queue (outside Influx).

**Comment by Benhgift** [comment]
@Igor-Toropov Yeah you can't count by time sadly, and even if you could, with over a billion datapoints, count takes hours to run in my tests. 

In postgres SQL this number is roughly tracked, so you can do a count estimate and it'll return instantly. I think influx should keep a running count of this too. 

I don't think it would be too hard to have an internal COUNT variable that gets incremented on every insert into a measurement.

**Comment by in-fke** [comment]
We would like to have some statistics for each measurement:

1. time of the first value, we do this with "select first(<field>) as <field> from <measurement>"
2. time of the last value, we do this with "select last(<field>) as <field> from <measurement>"
3. number of values, we do this with "select count(<field>) as count from <measurement>"

and currently 1/2 cannot be combined with 3 because count() apparently sets the timestamp to 0.
The above solution is very expensive and we haven't found a better one so far.
